lib/cylc/job_file.py

```diff
@@ -322,6 +322,6 @@ def _write_script(cls, handle, job_conf):
     @staticmethod
     def _write_epilogue(handle, job_conf):
         """Write epilogue."""
-        handle.write('\n\n. "${CYLC_DIR}/lib/cylc/job.sh"\ncylc__job__main')
+        handle.write('\n\n. "${CYLC_DIR}/job.sh"\ncylc__job__main')
```

Comment:
Perhaps we should move the job script to something like `bin/cylc-job.sh` - so it is available under the same location as `bin/cylc`. This should help get rid of `CYLC_DIR` fully.

Reply:
Sounds good to me. Is there any risk in doing so though? As this file would be put in the system or user `$PATH` alongside the other scripts.

Reply:
It should be fine. It should not be any riskier than, say, running the `cylc message` command.

Reply:
👍